### ISSUE TITLE
Bugsnag for tvOS use wrong path to write file

### ISSUE THREAD

**Issue by Alanko5** [bugsnag/bugsnag-cocoa#993]
[Bugsnag] [ERROR] Unable to write breadcrumb: Error Domain=NSCocoaErrorDomain Code=513 "Nemáte práva potrebné na uloženie súboru „4.json“ do priečinka „System“." UserInfo={NSURL=file:///4.json, NSUserStringVariant=Folder, NSUnderlyingError=0x2832790e0 {Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted"}}

**Comment by Alanko5** [comment]
in version 6.5.0 not show this error

**Comment by xander-jones** [comment]
Hey @Alanko5,
* which version of `bugsnag-cocoa` are you seeing this on? 
* which version of tvOS are you using?
* which device/simulator are you using?

The error is showing that the root directory could not be determined. This happens when Bugsnag's not able to create a directory: https://github.com/bugsnag/bugsnag-cocoa/blob/master/Bugsnag/Storage/BSGFileLocations.m#L22-L51

Are you seeing one of the following warnings in your console output:
```
Could not locate directory path for NSApplicationSupportDirectory.
```
or
```
Directory path for NSApplicationSupportDirectory is empty!
```

**Comment by xander-jones** [comment]
Hey @Alanko5, we've managed to reproduce this issue, and confirmed it's a bug. We'll take a look into a fix for this and let you know as soon as this is released.

In the meantime, I would suggest reverting to https://github.com/bugsnag/bugsnag-cocoa/releases/tag/v6.5.0 which you've noted is working as expected.

**Comment by phillipsam** [comment]
A fix for this has now been released in [v6.6.2](https://github.com/bugsnag/bugsnag-cocoa/releases/tag/v6.6.2). Let us know if you have any questions!

**Comment by clr182** [comment]
@joaopdcgarcia 
This issue does not seem to be related to the original post. 
The issue you are seeing also seems expected as there is no space left on the device to write to the json file.
